charts/opencost/templates/route.yaml

```
@@ -0,0 +1,32 @@
+{{- if and .Values.opencost.ui.enabled .Values.opencost.ui.route.enabled }}
```

```suggestion
{{- if and .Capabilities.APIVersions.Has "route.openshift.io/v1/Route" .Values.opencost.ui.enabled .Values.opencost.ui.route.enabled }}
```
Just to ensure that non-openshift users do not install it.
https://helm.sh/docs/chart_template_guide/builtin_objects/

---

not that much important though. the helm values clearly states that it is `openshift route` and disabled by default

---

Nevertheless added this.